1000genomes/README.md

```diff
@@ -24,6 +24,7 @@
  * [Exploring the phenotypic data](./data-stories/exploring-the-phenotypic-data)
  * [Exploring the variant data](./data-stories/exploring-the-variant-data)
  * [Understanding Alternate Alleles in 1,000 Genomes](./data-stories/understanding-alternate-alleles)
+ * [Data Analysis Codelab](https://github.com/googlegenomics/codelabs/tree/master/R/1000Genomes-BRCA1-analysis)
```

Comment:
Nicole, looking at the [following SQL script](https://github.com/googlegenomics/codelabs/blob/master/R/1000Genomes-BRCA1-analysis/sql/gwas-brca1-pattern.sql) are you running this across all populations?  BRCA1 susceptibility is not the same for all populations.  How did you pick P <= 5*10^-8?